packages/shared/src/contexts/FeaturesContext.tsx

```diff
@@ -89,7 +89,7 @@ const getFeatures = (flags: IFlags): FeaturesData => {
       Features.ScrollOnboardingVersion,
       flags,
     ),
-    hasSquadAccess: isFeaturedEnabled(Features.HasSquadAccess, flags),
+    hasSquadAccess: isFeaturedEnabled(Features.HasSquadAccess, flags) ?? true,
```

Comment:
Just tagging for visibility if it is intentional to be true.

Reply:
Phew! Thanks for noticing that! 🤯